mod_reforged/hooks/skills/effects/berserker_rage_effect.nut

```diff
@@ -0,0 +1,46 @@
+::mods_hookExactClass("skills/effects/berserker_rage_effect", function (o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+        this.m.Description = "This character is in a berserking rage."
+	}
+
+	o.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.extend([
+			{
+				id = 10,
+				type = "text",
+                icon = "ui/icons/special.png",
+				text = "Receives only " + ::MSU.Text.colorGreen((this.calcDamageTakenMult() * 100.0) + "%") + " of any damage"
```

Comment:
Better wording would be: All damage received is reduced by x%

Reply:
I know vanilla BF specifics `damage received` instead of `damage reduced` but based on player feedback over on the PTR server players like to see the reduction as it is much easier to understand.